### DISCUSSION TITLE
How to turn off mypy?

### DISCUSSION THREAD

**Discussion by igor-yusupov** [zed-industries/zed#20975] [Feature Requests]
I want to turn off mypy checkers after updates. How to do that?

**Comment by db-keli** [comment]
New updates seem to run with mypy and you can configure mypy with a pyproject.toml in your project directory
Example is this:
![Screenshot from 2024-11-21 14-46-48](https://github.com/user-attachments/assets/be8104d6-03c3-4a48-afdd-6114cf05d307)

to totally disable mypy you can explicitly disable mypy by adding no rules or by setting it to ignore all files. like this ;
```
[tool.mypy]
files=[]
```

**Reply by txstc55** [reply]
somehow doesn't work for me. I put the project.toml both at root, still got mypy linting
also how to turn off pycodestyle

**Comment by igor-yusupov** [comment]
It seems that there is a solution: https://github.com/zed-industries/zed/issues/20646

**Reply by txstc55** [reply]
I think they also pushed an update patch to turn it off as default, I guess many people are complaining.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
